src/main/fc/rc_controls.c

```diff
@@ -83,7 +83,6 @@ PG_REGISTER_WITH_RESET_TEMPLATE(armingConfig_t, armingConfig, PG_ARMING_CONFIG,
 
 PG_RESET_TEMPLATE(armingConfig_t, armingConfig,
     .gyro_cal_on_first_arm = 0,  // TODO - Cleanup retarded arm support
-    .disarm_kill_switch = 1,
```

Comment:
This change requires the PG version number to be incremented.

Reply:
oh. i thought pg version only get changed with releases. i will add it

Reply:
We are talking about different things here. What I meant was the version number of the parameter group the parameter got removed from:

https://github.com/wind0r/betaflight/blob/45b93a11883ca03c966bc9c08f0102dbb6fa47e2/src/main/fc/rc_controls.c#L82

(the number at the end)